forest/daemon/src/daemon.rs

```diff
@@ -481,6 +484,22 @@ fn get_actual_chain_name(internal_network_name: &str) -> &str {
     }
 }
 
+#[cfg(feature = "rocksdb")]
```

Comment:
Is it possible to avoid using orthogonal / mutually exclusive features? Maybe via --db [rocks | parity] cli option (in future PRs). Personally I don't like mutually exclusive features that makes `--all-features` not working